src/win32.c

```diff
@@ -1032,4 +1032,35 @@ gchar *win32_get_user_config_dir(void)
 	return g_build_filename(g_get_user_config_dir(), "geany", NULL);
 }
 
+
+/* Retrieve the console codepage
+ * In case GetConsoleOutputCP() returns 0 (i.e. the application doesn't have an own console window
+ * fallback to GetOEMCP(). */
+guint win32_get_console_codepage(void)
+{
+	guint codepage = GetConsoleOutputCP();
+	if (codepage == 0)
```

Comment:
Yeah, experience. No idea what it means but if I call `GetConsoleOutputCP` in Geany with a debug console (as if when started with `-v`) then the return value is 0 and hence the "fallback" to `GetOEMCP`.
Well, I assume I don't get a proper answer from `GetConsoleOutputCP` because there is no console at all for the Geany process and so it probably returns just 0. Though it's not documented, yes.